nuitka/ModuleRegistry.py

```diff
@@ -174,7 +174,7 @@ def getDoneModulesCount():
 
 
 def getDoneModules():
-    return sorted(done_modules, key=lambda module: (module.getFullName(), module.kind))
+    return sorted(done_modules, key=sort_key)
```

Comment:
I think it's ugly to have this as a separate function. Something the AI cannot come up with, but maybe modules need to implement a comparison themselves, but  this hurts readability in a non-performance critical function, I don't really want to look at a separate function. You only save the time it takes to create the lambda function, which cannot be that much.